starknet-rpc-test/get_storage_at.rs

```diff
@@ -0,0 +1,64 @@
+extern crate starknet_rpc_test;
+
+use starknet_core::types::{BlockId, StarknetError};
+use starknet_ff::FieldElement;
+use starknet_providers::ProviderError::StarknetError as StarknetProviderError;
+use starknet_providers::{MaybeUnknownErrorCode, Provider, StarknetErrorWithMessage};
+use starknet_rpc_test::{ExecutionStrategy, MadaraClient};
+
+#[tokio::test]
+async fn work_ok_at_storage() -> Result<(), anyhow::Error> {
+    let madara = MadaraClient::new(ExecutionStrategy::Native).await;
+    let rpc = madara.get_starknet_client();
+
+    // Return an error for a nonexisting block
+    let err = rpc
+        .get_storage_at(
+            FieldElement::from_hex_be("0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7").unwrap(),
```

Comment:
Wait for https://github.com/keep-starknet-strange/madara/pull/969 to be merged and use the const it defines insead

Reply:
ok, now merged